.github/workflows/build-and-test.yml

```diff
@@ -141,15 +141,16 @@ jobs:
   unittest-matrix:
     strategy:
       matrix:
-        runner: [ubuntu-latest]
```

Comment:
is this relevant to junit?

Reply:
i use matrix.os later on - and we use matrix.os in other files.  I chose to align the different files.

Reply:
Reversed this